app/views/layouts/_utility_menu.erb

```diff
@@ -3,17 +3,10 @@
     <%= utility_menu_link('grid_view', 'dashboard', 'dashboard', 'Dashboard') %>
     <% if current_user.role == "challenge_manager" %>
         <%= utility_menu_link('emoji_events', Rails.configuration.phx_interop[:phx_uri], 'challenges', 'Challenges') %>
-        <%= utility_menu_link('content_copy', 'evaluation_forms', 'Evaluation Forms', 'Evaluations') %>
-        <%= utility_menu_link('folder_open', 'manage_submissions', 'Manage Submissions', 'Submissions') %>
+        <%= utility_menu_link('check_circle_outline', 'evaluation_forms', 'Evaluation Forms', 'Evaluation Forms') %>
     <% end %>
-    <% if current_user.role == "evaluator" %>
-        <%= utility_menu_link('content_copy', 'evaluations', 'Evaluate Submissions', 'Evaluations') %>
-    <% end %>  
-    <% if current_user.role == "solver" %>
-        <%= utility_menu_link('folder_open', 'submissions', 'submissions', 'Submissions') %>
-    <% end %>
-    <%= utility_menu_link('map', 'user_guide', 'user guides', 'User Guides') %>
-    <%= utility_menu_link('support_agent', 'federal-agency-faqs', 'help', 'Help') %>
+    <%= utility_menu_link('star_half', 'manage_submissions', 'Manage Submissions', 'Submissions') %>
```

Comment:
this route needs to be role-specific, for evaluators it is `/evaluations`, while challenge managers have `/manage_submissions`. the conditional was correct before it should just update the icon.

Reply:
I had to fix the routes on the _utility_menu on your other PR #169 because they were broken on non-index routes. the routes needed the `/` prefix so I switched them to use routes helpers instead of just strings and it works now. just a heads up because it will cause conflicts when that is merged.